store/streaming/example_config.toml

```
@@ -0,0 +1,10 @@
+[store]
+    streamers = [ # if len(streamers) > 0 we are streaming
+        "file", # name of the streaming service, used by constructor
+    ]
+
+[streamers]
+    [streamers.file]
+        keys = ["list", "of", "store", "keys", "we", "want", "to", "expose", "for", "this", "streaming", "service"]
+        writeDir = "path to the write directory"
```

TOML standard is to use `_`

```suggestion
        write_dir = "path to the write directory"
```

---

Thanks, oof I've been doing the opposite for ages.